frontend/src/uxbox/main/ui/hooks.cljs

```diff
@@ -87,11 +87,52 @@
         (dissoc state :timer))
       state)))
 
+(def sortable-ctx (mf/create-context nil))
+
+(mf/defc sortable-container
+  [{:keys [children] :as props}]
+  (let [global-drag-end (rx/subject)]
```

Comment:
you should memoize with `mf/use-memo` the subject creation

Reply:
done!